frontend/src/assets/styles/global.css

```diff
@@ -33,3 +36,78 @@ nav {
 .rsprd-link:not(:last-child) {
   margin-right: 16px;
 }
+
+.rsprd-button {
```

Comment:
Nitpick: Two good common components to have are a button component and an input commponent. My recommendation is to make those common components and move these stylings (button and input) into them.

Reply:
will do, in separate PR